File: Project_02_C#/project_2_CSharp/project_2_CSharp/Customer/Customer.cs
using project_2_CSharp.personnel;
using System;
using System.Collections.Generic;
using System.Text;

namespace project_2_CSharp.Customer
{
    class Customer:Person
    {
        private string email;
        private string address;

        public string Email { get => email; set => email = value; }
        public string Address { get => address; set => address = value; }

        public Customer(){ }

        public Customer(string name, string cMND, int yearOfBirth, string email, string address) : base(name, cMND, yearOfBirth)
        {
            Email = email;
            Address = address;
        }

    }
}

File: project_2_java/src/customer/Customer.java
package customer;

import personnel.Person;

public class Customer extends Person
{
	private String email;
	private String address;

	public final String getEmail()
	{
		return email;
	}
	public final void setEmail(String value)
	{
		email = value;
	}
	public final String getAddress()
	{
		return address;
	}
	public final void setAddress(String value)
	{
		address = value;
	}

	public Customer()
	{
	}

	public Customer(String name, String cMND, int yearOfBirth, String email, String address)
	{
		super(name, cMND, yearOfBirth);
		setEmail(email);
		setAddress(address);
	}

}
